.editorconfig

```diff
@@ -0,0 +1,25 @@
+root = true
+
+[*]
+indent_style = space
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+charset = utf-8
+
+# Python files
+[*.py]
+indent_size = 4
+# isort plugin configuration
+known_first_party = sympy
+multi_line_output = 2
+default_section = THIRDPARTY
+skip = .eggs
+
+# RST files (used by sphinx)
+[*.rst]
+indent_size = 4
```

Comment:
I see both in the random file that I checked in the docs. If this should be 3, we should standardize the rst files and then change this setting with a new commit.